Cargo.toml

```diff
@@ -18,18 +18,18 @@ encoding_rs = "0.7"
 futures = "0.1.21"
 http = "0.1.5"
 hyper = "0.12.2"
-hyper-tls = "0.2.1"
+hyper-tls = "0.3"
 libflate = "0.1.11"
 log = "0.4"
 mime = "0.3.7"
 mime_guess = "2.0.0-alpha.4"
-native-tls = "0.1.5"
+native-tls = "0.2"
 serde = "1.0"
 serde_json = "1.0"
 serde_urlencoded = "0.5"
 tokio = "0.1.7"
 tokio-io = "0.1"
-tokio-tls = "0.1"
+tokio-core = "0.1"
```

Comment:
The `tokio-core` dependency should no longer be needed either. After that, I think we're good to go!

Reply:
Sorry :( Forgot this!